src/ott-analytics.js

```diff
@@ -231,7 +231,7 @@ class OttAnalytics extends BasePlugin {
    */
   get _eventParams(): Object {
     return {
-      mediaType: MEDIA_TYPE,
+      mediaType: this.player.config.sources.type || MEDIA_TYPE,
```

Comment:
The media type is not the source type, but the OTT media type value which is passed to the loadMedia method.
See `mediaType` options in [mediainfo structure](https://github.com/kaltura/kaltura-player-js/blob/master/docs/playing-your-video.md#mediainfo-structure-1).
Currently I don't think we pass them in anyway to the created source. 
Task is to add this info to the sources(maybe metadata) and use it in the plugin.

Reply:
Also, please add test for this change.

Reply:
I'm doing it now

Reply:
Pay attention - the `sources.type` wasn't planned for this value, it is only VOD or LIVE.
Need to see what this will cause(it is used by Kava for example)

Reply:
So what should be reported? the media format? like "TV Main"?